docker-compose.yml

```diff
@@ -15,8 +15,9 @@ services:
             - EVENTSTORE_RUN_PROJECTIONS=all
 
     postgres:
-        image: postgres:10-alpine
+        build: ./pgsql/
```

Comment:
that's not needed, you can use something like:
volumes:
- ./src/PdoEventStore/scripts/postgres:/docker-entrypoint-initdb.d